CRM/Activity/Form/Activity.php

```diff
@@ -523,7 +523,7 @@ public function preProcess() {
 
     if ($this->_action & CRM_Core_Action::UPDATE) {
       // We filter out alternatives, in case this is a stored e-mail, before sending to front-end
-      $this->_values['details'] = CRM_Utils_String::stripAlternatives($this->_values['details']);
+      $this->_values['details'] = CRM_Utils_String::stripAlternatives($this->_values['details']) ?: '';
```

Comment:
Huh, that's the first time I've encountered this type of expression in php. Looking it up - it's called the Elvis operator. Cool.
![image](https://user-images.githubusercontent.com/2874912/35124793-5f396a92-fc75-11e7-9f0e-d4744ca46c6e.png)

Reply:
lol

Reply:
In php7 you get a double ?? option to get around doing isset()